core/types/hashing.go

```diff
@@ -201,6 +201,22 @@ func prefixedRlpHash(prefix byte, x interface{}) (h libcommon.Hash) {
 	return h
 }
 
+// doublePrefixedRlpHash writes two prefixes into the hasher before rlp-encoding the
+// given interface. It's used for typed transactions.
+func doublePrefixedRlpHash(prefix1, prefix2 byte, x interface{}) (h libcommon.Hash) {
```

Comment:
Isn't this like the 10th function in the codebase doing this? Can't we reuse some of the already existing code?

Reply:
Can you point out another function that does the same thing? Was unable to find any, happy to use it if it already exists.